src/index.ts

```diff
@@ -10,10 +11,19 @@ type NotifyOptions = {
 const notify = async (
   title: string = DEFAULT_TITLE,
   description: string = DEFAULT_DESCRIPTION,
-  options: NotifyOptions,
+  options: NotifyOptions
 ) => {
+  if (PLATFORM === "linux") {
+    return await execa(`notify-send`, [
+      `-a`,
+      `NOTIFY-ME`,
+      `${title} ${description}`,
```

Comment:
If title and description be more of one word like

title = "This is a notification"
description "Example of description with multiple word"

How notify-send will be known where title finish?

Reply:
Hmm, if someone see a problem on that, we can try other solution in the future, for now is good enough.
Thank you so much for your contribution